code/modules/chemistry/Reagents-FoodDrink.dm

```diff
@@ -3759,3 +3759,77 @@ datum
 			alch_strength = 0.4
 			description = "A warm, late night drink, usually enjoyed during long winter nights."
 			reagent_state = LIQUID
+
+		fooddrink/alcoholic/grenadine
```

Comment:
Should probably be `fooddrink/grenadine` if it's not alcoholic?

Reply:
i was thinking the same, but it's also kinda only a cocktail ingredient so it's sorta like an alcoholic component? I don't really mind either way but that's the main train of thought I had when making this.

Reply:
eh, misleading paths aren't good imo